plugins/inputs/ping/ping_windows.go

```diff
@@ -34,9 +34,9 @@ func (p *Ping) pingToURL(u string, acc telegraf.Accumulator) {
 	if err != nil {
 		// fatal error
 		if pendingError != nil {
-			acc.AddError(fmt.Errorf("%s: %s", pendingError, u))
+			acc.AddError(fmt.Errorf("%q: %w", host, pendingError))
 		} else {
-			acc.AddError(fmt.Errorf("%s: %s", err, u))
+			acc.AddError(fmt.Errorf("%q: %w", host, err))
```

Comment:
When the host is the only value before the colon, why quote it?

Reply:
Right, changed.